test/ClangModules/objc_parse.swift

```diff
@@ -305,7 +305,7 @@ class NSObjectable : NSObjectProtocol {
 // Properties with custom accessors
 func customAccessors(_ hive: Hive, bee: Bee) {
   markUsed(hive.isMakingHoney)
-  markUsed(hive.makingHoney()) // expected-error{{cannot call value of non-function type 'Bool'}}
+  markUsed(hive.makingHoney()) // expected-error{{cannot call value of non-function type 'Bool'}}{{28-29=}}
```

Comment:
If I understand correctly, this fix-it only removes l-paren `(`. Shouldn't it be `{{28-30=}}`?